crates/biome_css_formatter/src/utils/component_value_list.rs

```diff
@@ -57,11 +60,16 @@ where
                             } else {
                                 write!(f, [space()])?;
                             }
+                        } else if at_group_boundary {
+                            write!(f, [hard_line_break()])?;
                         } else {
                             write!(f, [soft_line_break_or_space()])?
                         }
                     }
 
+                    at_group_boundary =
+                        is_comma && matches!(layout, ValueListLayout::OneGroupPerLine);
+
```

Comment:
You might want to add a comment that explains why we change the value of `at_group_boundary` **after** the first iteration of the loop.